### PR TITLE
[4.0] com_messages - remove unused submenu

### PR DESCRIPTION
**Pull request by brianteeman**
Remove unused references to addSubmenu in various views on com_messages

### CODE REVIEW

File: administrator/components/com_messages/Helper/MessagesHelper.php [hunk 1]
```diff
@@ -21,30 +21,6 @@
...
-	 * @param   string  $vName  The name of the active view.
-	 *
-	 * @return  void
-	 *
-	 * @since   1.6
-	 */
-	public static function addSubmenu($vName)
-	{
-		\JHtmlSidebar::addEntry(
-			Text::_('COM_MESSAGES_ADD'),
```
Comment (Quy) [inline administrator/components/com_messages/Helper/MessagesHelper.php:36]: Remove language strings.

**Comment by wilsonge** [comment]
Thanks

File: administrator/components/com_messages/Helper/MessagesHelper.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_messages/Helper/MessagesHelper.php:36]: It's used in the menu iirc

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**